optee-utee/optee-utee-sys/src/tee_api.rs

```diff
@@ -17,9 +17,9 @@
 
 use super::*;
 
-#[cfg(feature = "std")]
-use std::os::raw::*;
-#[cfg(not(feature = "std"))]
+//#[cfg(target_os = "optee")]
+//use libc::*;
+//#[cfg(not(target_os = "optee"))]
```

Comment:
These can be dropped as I only required then initially since `std` support required old rust toolchain which didn't provide `core::ffi::*;` which isn't the case any longer. Same for all other instances below.

Reply:
done